src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1483,6 +1488,14 @@ internal static void ApplyConfiguration(JObject config, ScriptHostConfiguration
                 }
             }
 
+            if (config.TryGetValue("lazyLoadExtensions", out value))
```

Comment:
I don't think much thought was given to that name - it was just chosen to convey intent. What do you think the right name should be? We should use what we think is best. I don't think "lazy" is correct, so I was just trying to come up with something better.